core/templates/domain/objects/objects-domain.constants.ts

```diff
@@ -64,4 +64,18 @@ export class ObjectsDomainConstants {
       base_unit: '0.01 rupee'
     }
   };
+  public static RATIO_PARSING_ERRORS = {
+    INVALID_COLONS:
+      'Your answer has two colons (:) next to each other.',
+    INVALID_CHARS:
+      'Please write a ratio that consists of digits separated by colons' +
+      '(e.g. 1:2 or 1:2:3).',
+    INVALID_FORMAT:
+      'Please enter a valid ratio (e.g. 1:2 or 1:2:3).',
+    INVALID_RATIO:
+      'For this question, each element in your ratio should be a whole' +
+      'numbers (not a fraction or a decimal).',
+    INVALID_FORM:
+      'It seems like the input can be reduced further.'
```

Comment:
This error seems to occur even if the rule is 'IsEqualsTo', which doesn't seem right? Since there may be cases where the creator can expect exactly equal to '2:4', and not 1:2 or 3:6.

Reply:
yes that's is correct because 2:4 and 1:2 both should be considered as a valid answer right?

Reply:
@aks681 can I exclude this check based upon the selected rule?

Reply:
Well, if the rule is 'ExactlyEqualTo', 1:2 and 3:6 should not be matched with 2:4.